integration/fixtures.go

```diff
@@ -0,0 +1,20 @@
+package integration
+
+var repoShow = `
```

Comment:
I would be very careful testing against this human readable text.
I think it is time to introduce a `--json` argument to the CLI and test against that.
The human readable stuff is likely to change, and we don't really want others to depend on
out text output if they base tools/scripts on out CLI.

Reply:
I would extend `github.com/humio/cli/prompt` to support JSON output and use that for all output.

Reply:
The tests themselves are fine. But if you plan on using drone, would you use `docker-compose` or a service in the .drone.yaml file?

Reply:
Good idea on the json output. Will look at that.

Reply:
I don't have much experience with drone. I will need to look into if docker-compose can be run as a drone task? or if this will need to be re-worked a bit. I do think we'll want to have it built/deployed via drone.